packages/react-core/src/components/Alert/Alert.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { css, getModifier } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import styles from '@patternfly/patternfly-next/components/Alert/styles.css';
+
+import AlertIcon from './AlertIcon';
+import AlertBody from './AlertBody';
+import AlertAction from './AlertAction';
+
+export const alertType = ['success', 'danger', 'warning', 'info'];
```

Comment:
let's make this an object
```
export const AlertVariant = {
  success: 'success',
  ....
};
```

That allows users to user AlertVariant.success instead of using string literals the array is not all that useful.
 UpperCamelCased since it is more of an enum, and matches the casing/implementation `ButtonVariant`

Reply:
That's a great point! I'll rewrite it.

I've also added function to capitalize first letter so we have easier way of writing sr-only and aria label.